outofcore/tools/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ else(NOT VTK_FOUND)
   include("${VTK_USE_FILE}")
   include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
   
-  if("${VTK_MAJOR_VERSION}.${VTK_MINOR_VERSION}" VERSION_GREATER "5.8")
+  if("${VTK_MAJOR_VERSION}.${VTK_MINOR_VERSION}" VERSION_GREATER "5.8" AND "${VTK_MAJOR_VERSION}.${VTK_MINOR_VERSION}" VERSION_LESS "7.0")
```

Comment:
I would rather check `VTK_RENDERING_BACKEND` so that it allows people with OpenGL backend to still use the out-of-core module.